model_analyzer/result/run_config_result_comparator.py

```diff
@@ -14,10 +14,6 @@
 
 from typing import List, Dict
 
-from model_analyzer.record.record import Record
-
-from model_analyzer.constants import COMPARISON_SCORE_THRESHOLD
```

Comment:
@nv-braf This went away because we used to only say A was less than B if it was at least 5% less? Now it is strictly less than?

Reply:
Correct.